src/App.js

```diff
@@ -1,25 +1,74 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import { Card, Row, Col, Divider, Input, Button } from 'antd';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import Searchbar from './components/Searchbar';
 
 function App() {
+  const [food, setFood] = useState(foods);
+  const [alwaysFood, setAlwaysFood] = useState(foods);
+  const [visible, setVisible] = useState(true);
+  const [isFoodArrayEmpty, setIsFoodArrayEmpty] = useState(false);
+
+  const addToList = (item) => {
+    const foodCopy = [item, ...food];
+
+    setFood(foodCopy);
+  };
+
+  const searchFood = (query) => {
+    const filteredFood = food.filter((item) =>
+      item.name.toLowerCase().includes(query.toLowerCase())
+    );
```

Comment:
Your search bar is not working because you're using `.includes()` that is an array method.

You want to compare `item.name` with `query` both are `strings` so `.includes()` will never work here unless you turn those `strings `into `arrays`.

Or you can simply use `.startsWith()` that is a method that compares two `strings`:
```
item.name.toLowerCase().startsWith(query.toLowerCase())
```